### PR TITLE
Allow more than 5 polls (but only 5 polls at a time still)

### PR DESCRIPTION
**Pull request by Lycanium-Z**

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
Theres probs a better way to do this. i dont think i did it properly, but @Volco the code should be smth like this.

File: chat-plugins/poll.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 
 'use strict';
-
+let activePolls = 0;
```
Comment (Volco) [inline chat-plugins/poll.js]: imo scrap the activePolls stuff (ill say how to fix the deletion correctly in a bit)

File: chat-plugins/poll.js [hunk 2]
```diff
@@ -362,6 +363,7 @@ exports.commands = {
 			if (room.poll.pollArray[num].pollNum === parseInt(target)) delete room.poll.pollArray[num];
 
 			return this.privateModCommand("(A poll was ended by " + user.name + ".)");
+			activePolls = activePolls - 1;
```
Comment (Volco) [inline chat-plugins/poll.js]: scrap the activePolls part and change the ``delete room.poll.pollArray[num]`` to this ``room.poll.pollArray.splice(num, 1)`` on like 363

File: chat-plugins/poll.js [hunk 3]
```diff
@@ -224,7 +224,7 @@ exports.commands = {
 			if (target.length > 1024) return this.errorReply("Poll too long.");
 
 			const supportHTML = cmd === 'htmlcreate';
-			if (room.poll && room.poll.pollArray.length >= 5) return this.errorReply('There can only be up to 5 polls at a time.');
+			if (room.poll && activePolls >= 5) return this.errorReply('There can only be up to 5 polls at a time.');
```
Comment (Volco) [inline chat-plugins/poll.js]: change this back

File: chat-plugins/poll.js [hunk 4]
```diff
@@ -276,6 +276,7 @@ exports.commands = {
 
 			this.roomlog("" + user.name + " used " + message);
 			return this.privateModCommand("(A poll was started by " + user.name + ".)");
+			activePolls = activePolls + 1;
```
Comment (Volco) [inline chat-plugins/poll.js]: get rid of this

File: chat-plugins/poll.js (hunk 2 above)
Comment (Volco) [reply chat-plugins/poll.js]: Also do this for timer on line 326 (it wont let me review there kek)

File: wavelength-plugins/survey.js [hunk 5]
```diff
@@ -8,6 +8,7 @@
 ***********************************/
 'use strict';
 
+let actSurveys = 0;
```
Comment (Volco) [inline wavelength-plugins/survey.js]: do the same stuff for surveys that i mentioned for polls

**Comment by Lycanium-Z** [comment]
fine volco i fix it

**Comment by Volco** [comment]
hooray (you'll thank me later when you are better at coding lol)

**Pull request merged**